src/layers/src/base-layer.ts

```diff
@@ -1428,6 +1428,16 @@ class Layer {
     // implemented in subclasses
     return () => null;
   }
+
+  getColumnAccessor(dataContainer?: DataContainerInterface): (...args: any[]) => any {
```

Comment:
Do the return types need to be `any`? Can we be more precise?

Reply:
I think we don't need these placeholders in base-layer. The geoColumnAccessor and geoFieldAccessor in geojson-layer should work without these placeholders. I will try to consolidate them if needed when supporting arrow format on other layers.